curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657e6231622cd919e2d7094b.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 657e6231622cd919e2d7094b
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+Now it's time to be able to append each expense as a dictionary to the `expenses` list. This illustrates how to create functions that manipulate lists by adding new items.
+
+
+To append a value to a list, the `append` methos is used. This is the syntax used to append a value to a list:
+```python
+def function_name(parameter_name):
+    list_name = []
+    list_name.append(parameter_name)
+```
```

Comment:
```suggestion

```python
def function_name(parameter_name):
    list_name = []
    list_name.append(parameter_name)
```
```

Reply:
I would suggest adding a blank line before the example.